modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -306,15 +312,165 @@ const SaveFDSFieldsModalContent = ({
 	);
 };
 
+interface EditFDSFieldModalContentProps {
+	closeModal: Function;
+	fdsField: FDSField;
+	namespace: string;
+	onSave: Function;
+}
+
+const EditFDSFieldModalContent = ({
+	closeModal,
+	fdsField,
+	namespace,
+	onSave,
+}: EditFDSFieldModalContentProps) => {
+	const [selectedFDSFieldRenderer, setSelectedFDSFieldRenderer] = useState(
+		fdsField.renderer ?? 'default'
+	);
+	const [fdsFieldSortable, setFSDFieldSortable] = useState<boolean>(
+		fdsField.sortable ?? true
+	);
+
+	const fdsFieldLabelRef = useRef<HTMLInputElement>(null);
+
+	const editFDSField = async () => {
+		const body = {
+			label: fdsFieldLabelRef.current?.value,
+			renderer: selectedFDSFieldRenderer,
+			sortable: fdsFieldSortable,
+		};
+
+		const response = await fetch(
+			`${API_URL.FDS_FIELDS}/by-external-reference-code/${fdsField.externalReferenceCode}`,
+			{
+				body: JSON.stringify(body),
+				headers: {
+					'Accept': 'application/json',
+					'Content-Type': 'application/json',
+				},
+				method: 'PATCH',
+			}
+		);
+
+		if (!response.ok) {
+			openToast({
+				message: Liferay.Language.get(
+					'your-request-failed-to-complete'
+				),
+				type: 'danger',
+			});
+		}
+
+		const editedFDSField = await response.json();
+
+		closeModal();
+
+		openToast({
+			message: Liferay.Language.get(
+				'your-request-completed-successfully'
+			),
+			type: 'success',
+		});
+
+		onSave({editedFDSField});
+	};
+
+	return (
+		<>
+			<ClayModal.Header>
+				{Liferay.Util.sub(
+					Liferay.Language.get('edit-x'),
+					fdsField.label
+				)}
+			</ClayModal.Header>
+
+			<ClayModal.Body>
+				<ClayForm.Group>
+					<label htmlFor={`${namespace}fdsFieldNameInput`}>
+						{Liferay.Language.get('name')}
+					</label>
+
+					<ClayInput
+						disabled
+						id={`${namespace}fdsFieldNameInput`}
+						type="text"
+						value={fdsField.name}
+					/>
+				</ClayForm.Group>
+
+				<ClayForm.Group>
+					<label htmlFor={`${namespace}fdsFieldLabelInput`}>
```

Comment:
Rather than declaring `${namespace}fdsFieldLabelInput` twice, once here and once for the input, it might make more sense to set as a variable.